domain/schema/controller/sql/0009-controller-node.sql

```diff
@@ -23,3 +23,20 @@ CREATE TABLE controller_node_agent_version (
     FOREIGN KEY (architecture_id)
     REFERENCES architecture (id)
 );
+
+CREATE TABLE controller_api_address (
+    controller_id TEXT NOT NULL,
+    -- The value of the configured IP address.
+    -- e.g. 192.168.1.2 or 2001:db8:0000:0000:0000:0000:0000:00001.
+    address_value TEXT NOT NULL,
+    -- one of ipv4, ipv6 etc.
+    type_id INT NOT NULL,
```

Comment:
This should be a FK to an enumeration table.

Reply:
All these constants (so the same answer applies to the other FK comments) are defined in the model db, and as I understood we wanted this table to have denormalized values that we construct with the new worker. The watcher will be responsible of updating the space uuid that it would retrieve from the model db. 
I'm happy to duplicate the scope and type constants on the controller db if I misunderstood.

Reply:
After @manadart 's required changes, this is no longer applicable, thanks for the catch.